hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/artifacts/smtable/SoftwareModuleAddUpdateWindow.java

```diff
@@ -262,9 +279,10 @@ private void populateValuesOfSwModule() {
         final SoftwareModule swModle = softwareManagement.findSoftwareModuleById(baseSwModuleId);
         nameTextField.setValue(swModle.getName());
         versionTextField.setValue(swModle.getVersion());
-        vendorTextField.setValue(swModle.getVendor() == null ? HawkbitCommonUtil.SP_STRING_EMPTY
+
+        vendorTextField.setValue(swModle.getVendor() == null ? StringUtils.EMPTY
```

Comment:
Is this necessary, because now all `null` value are shown as empty String. So think this is redundant.